metrics/density/README.md

```diff
@@ -0,0 +1,53 @@
+# Kata Containers density metrics tests
+
+* [Kata Containers density metrics tests](#kata-containers-density-metrics-tests)
+   * [docker_memory_usage](#docker_memory_usage)
+   * [fast_footprint](#fast_footprint)
+   * [footprint_data](#footprint_data)
+   * [memory_usage_inside_container](#memory_usage_inside_container)
+
+This directory contains a number of tests to help measure container
+memory footprint. Some measures are based around the
+[PSS](https://en.wikipedia.org/wiki/Proportional_set_size) of the runtime
+components, and others look at the system level (`free` and `/proc/meminfo`
+for instance) impact.
+
+## `docker_memory_usage`
+
+This test measures the PSS footprint of the runtime components whilst
+launching a number of small ([busybox](https://hub.docker.com/_/busybox/)) containers.
+
+## `fast_footprint`
+
+This test takes system level resource measurements after launching a number of
+containers in parallel and optionally waiting for KSM to settle its memory
+compaction cycles.
+
+The script is quite configurable via environment variables on which container
```

Comment:
Try a bulleted list for the options in this paragraph.

Reply:
I think I see what you mean @intelkevinputnam - have broken into a list and worded as such. ptal.